include/bbp/sonata/report_reader.h

```diff
@@ -23,7 +23,8 @@ struct SONATA_API DataFrame {
     std::vector<double> times;
     DataType ids;
     // data[times][ids]
-    std::vector<std::vector<float>> data;
+    size_t n_cols, n_rows;
```

Comment:
`n_rows` could be extracted from the data.size() (/n_cols) to avoid inconsistencies